Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM adoptopenjdk:8-jre-hotspot as builder
+FROM eclipse-temurin:17-jre as builder
```

Comment:
can we not use open jdk as well for Java 17 ?

Reply:
I tried to look for an openjdk image but [they are deprecated now](https://github.com/docker-library/openjdk/issues/505), and one of their suggestions is [Eclipse Temurin](https://hub.docker.com/_/eclipse-temurin/).

Reply:
makes sense. thanks!